From c5c42a3def02562235289d21c779103911c4e37b Mon Sep 17 00:00:00 2001
From: Sveinung Kvilhaugsvik <sveinung84@users.sourceforge.net>
Date: Mon, 2 Nov 2015 18:41:37 +0100
Subject: [PATCH] Europe 1901: remove intra paragraph line breaks

from the description. Adapting a paragraph of text to the screen used to
view it should be the client's job. (Horizontal space can be more valuable
than vertical space)
---
 data/scenarios/europe_1901.sav | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/scenarios/europe_1901.sav b/data/scenarios/europe_1901.sav
index c04a478..34dfa40 100644
--- a/data/scenarios/europe_1901.sav
+++ b/data/scenarios/europe_1901.sav
@@ -1,7 +1,7 @@
 
 [scenario]
 name=_("Europe 1901")
-description=_("Europe 1901\n\nAn attempt at an historically accurate recreation of\nthe Great Powers of Europe just prior to the outbreak\nof the First World War.\n\nBy Ferdinand Steinkrüger (a.k.a XYZ) and Jamie Troini (a.k.a. Nimrod).")
+description=_("Europe 1901\n\nAn attempt at an historically accurate recreation of the Great Powers of Europe just prior to the outbreak of the First World War.\n\nBy Ferdinand Steinkrüger (a.k.a XYZ) and Jamie Troini (a.k.a. Nimrod).")
 players=TRUE
 
 [savefile]
-- 
2.1.4

